Assignment 1/src/main/java/com/cezaram28/Assignment1/repository/jdbc/AnswerMapper.java

```diff
@@ -0,0 +1,19 @@
+package com.cezaram28.Assignment1.repository.jdbc;
+
+import com.cezaram28.Assignment1.entity.Answer;
+import com.cezaram28.Assignment1.entity.Question;
+import com.cezaram28.Assignment1.entity.User;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class AnswerMapper implements RowMapper {
```

Comment:
RowMapper is a generic interface. You should implement RowMapper<Answer> such that your code is type-safe.